platformio.ini

```diff
@@ -0,0 +1,14 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:nodemcu-32s]
+platform = espressif32
+board = nodemcu-32s
+framework = arduino
```

Comment:
Also recommend adding the following:
upload_speed = 921600
monitor_speed = 115200

The upload speed increase will make it way faster to flash. Setting a monitor speed makes it easier to know what speed you should output serial as. 

Note that you can change the CPU speed from the default 160MHZ up to 240MHZ if you need to using:
board_build.f_cpu = 240000000L

This will increase power consumption and increase performance (probably unnecessary)

Reply:
I think I'll keep the CPU speed as is for now but will for sure add the first two!